coding/coding_tests/multilang_utf8_string_test.cpp

```diff
@@ -2,49 +2,65 @@
 
 #include "coding/multilang_utf8_string.hpp"
 
-#include "3party/utfcpp/source/utf8.h"
+#include "base/control_flow.hpp"
 
+#include "3party/utfcpp/source/utf8.h"
 
 namespace
 {
-  struct lang_string
-  {
-    char const * m_lang;
-    char const * m_str;
-  };
+struct lang_string
+{
+  char const * m_lang;
+  char const * m_str;
+};
 
-  void TestMultilangString(lang_string const * arr, size_t count)
+lang_string gArr[] = {{"default", "default"},
+                      {"en", "abcd"},
+                      {"ru", "\xD0\xA0\xD0\xB0\xD1\x88\xD0\xBA\xD0\xB0"},
+                      {"be", "\xE2\x82\xAC\xF0\xA4\xAD\xA2"}};
+
+struct LangChecker
+{
+  LangChecker() = default;
+
+  base::ControlFlow operator()(char lang, string const & utf8s)
   {
-    StringUtf8Multilang s;
+    TEST_EQUAL(lang, StringUtf8Multilang::GetLangIndex(gArr[m_index].m_lang), ());
+    TEST_EQUAL(utf8s, gArr[m_index].m_str, ());
+    ++m_index;
+    return base::ControlFlow::Continue;
```

Comment:
If the lambda always returns Continue, why not to make it void?

Reply:
done